doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -504,6 +504,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0
+
+KRKNWK-19915: Total operational hours are saved to NVM with interval defined by :kconfig:option:`CONFIG_CHIP_OPERATIONAL_TIME_SAVE_INTERVAL` Kconfig option.
```

Comment:
I wonder, if we ca call it differently. If i would be the customer i might ignore it.

We should mention in the name that "Certification test case TC-DGGEN-2.1 in the test step 10d (tbc) fails when Total Operational Hours feature is enabled" or similar.

Also in the description could we elaborate a bit, that this attribute is optional, and customer is affected only when enables  the xxx PICS.

Reply:
Expanded the description and added a note that feature is optional